datarepo-clienttests/src/main/java/scripts/testscripts/BulkLoad.java

```diff
@@ -22,7 +22,7 @@
   /** Public constructor so that this class can be instantiated via reflection. */
   public BulkLoad() {
     super();
-    manipulatesKubernetes = true; // this test script manipulates Kubernetes
+    // manipulatesKubernetes = true; // this test script manipulates Kubernetes
```

Comment:
Commented code

Reply:
Thanks for catching that. I still cannot run the tool in my environment without disabling manipulateKubernetes...